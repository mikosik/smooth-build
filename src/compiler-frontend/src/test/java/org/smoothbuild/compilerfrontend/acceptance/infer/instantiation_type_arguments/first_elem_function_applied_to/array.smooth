@Native("impl.met")
A firstElement([A] array);
myValue = firstElement(["abc"]);
