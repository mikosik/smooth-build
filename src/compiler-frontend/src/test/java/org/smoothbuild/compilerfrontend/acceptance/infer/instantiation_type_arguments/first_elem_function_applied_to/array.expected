@Native("impl.met")
A firstElement<A>([A] array);
String myValue<>
  = firstElement<String>(["abc"]);
